core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/search/domain/SearchCriteria.java

```diff
@@ -20,8 +20,7 @@
 import org.broadleafcommerce.core.catalog.domain.Category;
 import org.broadleafcommerce.core.search.service.solr.SolrHelperService;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Revert this change and update the code formatter in your IDE to not squash imports into a wildcard import in order to be consistent with Broadleaf's code formatting guidelines